packages/react-dom/src/events/ReactDOMEventListener.js

```diff
@@ -259,20 +250,9 @@ export function removeTrappedEventListener(
   capture: boolean,
   listener: any => void,
   passive: void | boolean,
-) {
-  if (listener.remove != null) {
-    listener.remove();
-  } else {
-    const rawEventName = getRawEventName(topLevelType);
-    if (passiveBrowserEventsSupported) {
-      targetContainer.removeEventListener(rawEventName, listener, {
-        capture,
-        passive,
```

Comment:
The passive flag [is not needed when removing](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener#Matching_event_listeners_for_removal). Only the capture flag needs to be specified.